index/upside_down/index_reader.go

```diff
@@ -125,10 +105,9 @@ func (i *IndexReader) DocumentFieldTermsForFields(id index.IndexInternalID, fiel
 	fieldsMap := make(map[uint16]string, len(fields))
 	for _, f := range fields {
 		id, ok := i.index.fieldCache.FieldNamed(f, false)
-		if !ok {
-			return nil, fmt.Errorf("Field %s was not found in cache", f)
```

Comment:
Yeah, I added the error and eventually regretted it. It should just add fields that exist as you do.